README.md

```diff
@@ -1,120 +1,209 @@
-* `DuckDuckGo`: Most recent results from [DuckDuckGo.com](https://duckduckgo.com/)
-* `Facebook` : Returns the latest posts on a page or profile on [Facebook](https://facebook.com/) (There is an [issue](https://github.com/RSS-Bridge/rss-bridge/issues/2047) for public instances)
-* `FlickrExplore` : [Latest interesting images](https://www.flickr.com/explore) from Flickr
-* `GoogleSearch` : Most recent results from Google Search
-* `Identi.ca` : Identica user timeline (Should be compatible with other Pump.io instances)
-* `Instagram`: Most recent photos from an Instagram user (It is recommended to [configure](https://rss-bridge.github.io/rss-bridge/Bridge_Specific/Instagram.html) this bridge to work)
-* `OpenClassrooms`: Lastest tutorials from [openclassrooms.com](https://openclassrooms.com/)
-* `Pinterest`: Most recent photos from user or search
-* `ScmbBridge`: Newest stories from [secouchermoinsbete.fr](https://secouchermoinsbete.fr/)
-* `ThePirateBay` : Returns the newest indexed torrents from [The Pirate Bay](https://thepiratebay.se/) with keywords
-* `Twitter` : Return keyword/hashtag search or user timeline
-* `Wikipedia`: highlighted articles from [Wikipedia](https://wikipedia.org/) in English, German, French or Esperanto
-* `YouTube` : YouTube user channel, playlist or search
+![Screenshot #2](/static/twitter-rasmus.png?raw=true)
 
-And [many more](bridges/), thanks to the community!
+[Documentation](https://rss-bridge.github.io/rss-bridge/index.html)
 
-Output format
-===
+Check out RSS-Bridge right now on https://rss-bridge.org/bridge01 or find another
+[public instance](https://rss-bridge.github.io/rss-bridge/General/Public_Hosts.html).
 
-RSS-Bridge is capable of producing several output formats:
+## Tutorial
 
-* `Atom` : Atom feed, for use in feed readers
-* `Html` : Simple HTML page
-* `Json` : JSON, for consumption by other applications
-* `Mrss` : MRSS feed, for use in feed readers
-* `Plaintext` : Raw text, for consumption by other applications
+### Install with git:
 
-You can extend RSS-Bridge with your own format, using the [Format API](https://rss-bridge.github.io/rss-bridge/Format_API/index.html)!
+```bash
+cd /var/www
+git clone https://github.com/RSS-Bridge/rss-bridge.git
 
-Screenshot
-===
+# Give the http user write permission to the cache folder
+chown www-data:www-data /var/www/rss-bridge/cache
 
-Welcome screen:
+# Optionally copy over the default config file
+cp config.default.ini.php config.ini.php
 
-![Screenshot](/static/screenshot_rss-bridge_welcome.png?raw=true)
+# Optionally copy over the default whitelist file
+cp whitelist.default.txt whitelist.txt
+```
 
-RSS-Bridge hashtag (#rss-bridge) search on Twitter, in Atom format (as displayed by Firefox):
+Example config for nginx:
 
-![Screenshot](/static/screenshot_twitterbridge_atom.png?raw=true)
+```nginx
+# /etc/nginx/sites-enabled/rssbridge
+server {
+    listen 80;
+    # server_name example.com;
```

Comment:
should also include ipv6 variant
```
listen [::]:80;
```

Reply:
A user with ipv6 knowledge would know to do that. This minimal config is fine.